src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -406,10 +406,10 @@ public void verifyUserLoggedIn(final String userName) {
       } else {
         WebElement avatar = wait.forElementVisible(By.cssSelector(LOGGED_IN_USER_SELECTOR_OASIS));
         String loggedInUserName = avatar.getAttribute("alt");
-        if (!loggedInUserName.equals(userName) && !loggedInUserName.equals(userName + " avatar")) {
-          throw new IllegalArgumentException(
-              "Invalid user, expected " + userName + ", but found: " + loggedInUserName);
-        }
+//        if (!loggedInUserName.equals(userName) && !loggedInUserName.equals(userName + " avatar")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)